Content.Server/Tesla/Components/LightningSparkingComponent.cs

```diff
@@ -0,0 +1,27 @@
+using Content.Server.Tesla.EntitySystems;
+using Robust.Shared.Audio;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Server.Tesla.Components;
+
+/// <summary>
+/// The component changes the visual of an object after it is struck by lightning
+/// </summary>
+[RegisterComponent, Access(typeof(LightningSparkingSystem))]
+public sealed partial class LightningSparkingComponent : Component
+{
+    /// <summary>
+    /// Spark duration.
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public float LightningTime = 4;
```

Comment:
have field be `TimeSpan` and just use `LightningTime.TotalSeconds`, makes it clearer what unit

Reply:
I didn't quite get the second part